Source/WebCore/html/shadow/attachmentElementShadow.css

```diff
@@ -105,10 +111,14 @@ div#attachment-information-block {
 }
 
 div#attachment-action {
+#if (defined(WTF_PLATFORM_VISION) && WTF_PLATFORM_VISION)
+    font-size: x-small;
+#else
+    font-size: small;
+#endif
```

Comment:
Not new to your patch, but I feel like these should probably use dynamic type?

@squelart are the existing font sizes to spec?

Reply:
I opened a bug to convert the fonts to use -apple-system- ones fwiw